src/helpers/network/prettifyContractName.ts

```diff
@@ -1,5 +1,5 @@
 import { utils } from 'ethers'
-import truncateMiddleIfNeeded from 'helpers/truncateMiddleIfNeeded'
+import truncateMiddleIfNeeded from 'helpers/network/truncateMiddleIfNeeded'
```

Comment:
Why did you place it into `network`?

Reply:
Typically this function is used with addresses